quickwit/quickwit-opentelemetry/src/otlp/mod.rs

```diff
@@ -45,6 +44,22 @@ pub use trace::{
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct TraceId([u8; 16]);
 
+impl Serialize for TraceId {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let b64trace_id = BASE64_STANDARD.encode(self.0);
```

Comment:
why is it better there?
has anything else changed?

Reply:
It's the same code, I moved it unintentionally